sites/website/src/docs/design-systems/overview.md

```diff
@@ -0,0 +1,64 @@
+---
+id: overview
+title: What is the Design System?
+sidebar_label: Overview
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/overview.md
+---
+A *Design System* is somewhat of an amorphous term but can generally be thought of as a collection of resources for interactive media that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems generally manifest as component libraries surrounded by usage guidance and design principles.
+
+In code, FAST currently uses "Design System" in a narrower sense than the above; the Design System is a dictionary of data that informs the visual (and sometimes functional) representation of UI elements. The Design System captures, stores, and reflects any information that should inform the rendering of a UI element. This data is often referred to as *Design Tokens*. Common examples of data in a Design System are:
```

Comment:
What about... "FAST uses "Design System" in a narrower sense; the Design System is a dictionary of data that informs the visual (and sometimes functional) representation of UI elements."  

basically, I'm suggesting a more concise version.